pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.0'
+lockfileVersion: '6.1'
```

Comment:
I think you need to update pnpm version on your machine (`pnpm add -g pnpm`) - the latest is `8.6.7`. In one of the newer versions the `lockfileVersion` was downgraded from 6.1 to 6.0 ([pnpm PR](https://github.com/pnpm/pnpm/pull/6664)) so we should keep it at 6.0 here, too.

Reply:
oh, thanks for catching that!